core/templates/dev/head/services/ContextService.js

```diff
@@ -13,45 +13,55 @@
 // limitations under the License.
 
 /**
- * @fileoverview Service for returning information about an exploration's
+ * @fileoverview Service for returning information about a page's
  * context.
  */
 
 oppia.constant('PAGE_CONTEXT', {
-  EDITOR: 'editor',
-  LEARNER: 'learner',
+  EXPLORATION_EDITOR: 'editor',
+  EXPLORATION_PLAYER: 'learner',
+  QUESTION_EDITOR: 'question_editor',
   OTHER: 'other'
 });
 
 oppia.constant('EDITOR_TAB_CONTEXT', {
```

Comment:
Slightly confused about this change. Shouldn't it be EXPLORATION_EDITOR_TAB_CONTEXT with EDITOR and PREVIEW subfields (where the subfields represent the tabs)?

Reply:
In the previous PR you reviewed [this](https://github.com/oppia/oppia/pull/5151#discussion_r197662019) change. Although this makes more sense.

Reply:
(Also, sorry, I did get this wrong in the previous review! My apologies.)

Reply:
Yes, makes sense. Changed